lib/whois/parsers/whois.nic.me.rb

```diff
@@ -8,6 +8,7 @@
 
 
 require_relative 'base_afilias'
+require 'whois/scanners/whois.nic.me.rb'
```

Comment:
Any reason to not use a `BaseIcannCompliant` parser?

Reply:
Sorry, but I cannot remember, this was one and a half year ago and I am not using this gem anymore.